Bindings/Java/Matlab/Dynamic_Walker_Example/RunForwardAnalysis.m

```diff
@@ -0,0 +1,58 @@
+import org.opensim.modeling.*
+
+% Open Model
+model= Model('../Model/WalkerModel.osim');
+
+% Add Analyses to the Model
+forceReporter = ForceReporter();
+model.addAnalysis(forceReporter);
```

Comment:
Conflicts with name of script. Perhaps rename script to "RunForwardSimulation.m"?